### ISSUE TITLE
Debugger support

### ISSUE THREAD

**Issue by surendrajat** [LoyieKing/Smalise#13]
It may be overkill at the moment but is this a sane feature-request/idea for this plugin?
Ref: https://github.com/JesusFreke/smalidea

**Comment by surendrajat** [comment]
One more update on this one: https://github.com/CalebFenton/simplify/releases/tag/v1.3.0

**Comment by Gh0u1L5** [comment]
Debugger is definitely a great feature to have, I'm just wondering maybe before adding more fancy features, we need to firstly do the language server stuff as discussed in https://github.com/LoyieKing/Smalise/pull/4. Otherwise it will be more and more work to do when we refactor the plugin to a language server.

**Comment by surendrajat** [comment]
Agree. Let's keep it as a reminder. I think the language-server should be developed independently of the extension though(preferably standalone binary). It will ensure wider integration and contributions, perhaps. Then it can be shipped with the extension or can be downloaded as per the os architecture post-installation.

**Comment by surendrajat** [comment]
Since https://github.com/Surendrajat/APKLab is usable now and will ship Smalise as a dependency in next release, any extra feature to support debugging like making app debuggable, rebuilding, singing, and installing are already covered. I'm thinking to try out partial debugging with java source(from Jadx) in APKLab but debugging at smali level will be more complete and powerful.
